src/scenes/AccessionsRouter/edit/EndDryingReminderModal.tsx

```diff
@@ -56,45 +53,28 @@ export default function EndDryingReminderModal(props: EndDryingReminderModalProp
     }
   };
 
-  const onEnable = (id: string, value: boolean) => {
-    setEnable(value);
-    if (!value) {
-      setDate(undefined);
-      onChange('dryingEndDate', undefined);
-    }
-  };
-
-  const validate = (dateValue: string | undefined): boolean => {
+  const validateDate = (newValue?: string | number | undefined) => {
     setDateError('');
 
-    if (!dateValue) {
-      if (!enable) {
-        return true;
-      } else {
-        setDateError(strings.REQUIRED_FIELD);
-        return false;
-      }
-    }
-
-    const dateMs = new Date(dateValue).getTime();
-    const today = new Date(getTodaysDateFormatted()).getTime();
-
-    if (isNaN(dateMs)) {
+    // Invalid dates or blank entries will remove the reminder date
+    if (!newValue) {
       setDateError(strings.INVALID_DATE);
-      return false;
-    } else if (dateMs < today) {
+      return true;
```

Comment:
Remove the date error on line 61?

Reply:
I originally left it as a visual indicator that the date isn't formatted correctly, like if someone was trying to type in a date.  Do you think it would be better without?

Reply:
Got it, we need to show an error in that case.
Some thoughts, if it makes sense.
We could allow the user to clear the date and save, without showing an error. Essentially removing the drying reminder.
If the date is non-empty and invalid, we could show the error and prevent save.

Reply:
That was originally what I wanted to do, but the MUI datepicker version that we use doesn't provide easy access to the raw input data.  If/when we update MUI to 6.x there are a lot of improvements to datepicker that we could utilize.
I think in the meantime this isn't a bad compromise considering it's a corner case.